app/models/shipit/commit.rb

```diff
@@ -102,6 +102,12 @@ def self.from_github(commit)
         additions: commit.stats&.additions,
         deletions: commit.stats&.deletions,
       )
+
+      if record.pull_request?
+        record.pull_request_head_sha = commit.parents.last.sha
```

Comment:
This doesn't work with GitHub's squashed merges right? I don't quite remember if we handle them at all today.

Reply:
Good question, I will check.

Reply:
No, it does not work for squashed merged - but we already do not detect these as pull requests anyway, so we aren't e.g. creating `CommitDeployment`s for these PRs etc. We could fix this (it's not a big deal at Shopify - squashed merges are not preferred - but it may be elsewhere), but I would prefer to do that out of scope of the PR.